pkg/controller/util/node/controller_utils.go

```diff
@@ -118,13 +129,29 @@ func SetPodTerminationReason(kubeClient clientset.Interface, pod *v1.Pod, nodeNa
 	return updatedPod, nil
 }
 
+func MarkAllPodsNotReadyInMutliPartitions(kubeClients []clientset.Interface, node *v1.Node) error {
```

Comment:
nit: MarkAllPodsNotReadyInMutliPartitions" -> "MarkAllPodsNotReadyInPartitions"

"Multi" is not needed when Partition is in plural

Reply:
sure, will update

Reply:
updated.